cropper/app/lib/Crops.scala

```diff
@@ -21,8 +21,8 @@ object Crops {
   import scala.concurrent.ExecutionContext.Implicits.global
   import Files._
 
-  def outputFilename(source: SourceImage, bounds: Bounds, outputWidth: Int, isMaster: Boolean = false): String = {
-    s"${source.id}/${Crop.getCropId(bounds)}/${if(isMaster) "master/" else ""}$outputWidth.jpg"
+  def outputFilename(source: SourceImage, bounds: Bounds, outputWidth: Int, exportType: String, isMaster: Boolean = false): String = {
+    s"${source.id}/${Crop.getCropId(bounds)}/$exportType/${if(isMaster) "master/" else ""}$outputWidth.jpg"
```

Comment:
As spoken about over IM - I can't see the harm in this, but  I can't see why it would be beneficial to have this?

Potentially if we wanted to differentiate between a full-size, freeform crop and a full export, this might be useful, but I would argue those are the same thing.